### PR TITLE
Add Github Actions for test cases

### PR DESCRIPTION
**Pull request by mdeegen**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
```
Comment (boeddeker) [inline .github/workflows/tests.yml:15]: Can you add 3.8?

Comment (mdeegen) [reply .github/workflows/tests.yml:15]: In the azure-pipelines.yml it said "Python 3.8 does not work at the moment." so i did not add 3.8.
But if you say i should add it here i can do that, sure.

**Comment by jensheit** [comment]
the combination of two independent changes is a bit confusing here, but overall it seems to be working

**Review by jensheit — APPROVED** [review]

**Review by boeddeker — APPROVED** [review]
Thank you. Looks good to me.

**Pull request merged**